src/routes/version.js

```diff
@@ -1,16 +1,38 @@
 'use-strict';
 
 const Joi = require('joi');
-const { checkUpdate } = require('../controllers/versionController');
+const { shouldUpdate } = require('../controllers/versionController');
 
 exports.plugin = {
   name: 'routes-version',
   register: async (server) => {
     server.route({
       method: 'GET',
       path: '/check-update',
-      options: { auth: false, validate: { query: Joi.object({ apiVersion: Joi.string().required() }) } },
-      handler: checkUpdate,
+      options: {
+        auth: false,
+        validate: {
+          query: Joi.object({
+            apiVersion: Joi.string(),
+            mobileVersion: Joi.string(),
```

Comment:
quitte a faire 2 routes tu peux simplifier les query ici : 
- pour check-update, on n'enverra que `apiVersion`
- pour should-update, on n'enverra que `mobileVersion`

Reply:
done